ci-operator/step-registry/telcov10n/functional/ocp-cnf-tests/telcov10n-functional-ocp-cnf-tests-ref.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "telcov10n/functional/ocp-olm-setup/telcov10n-functional-ocp-cnf-tests-ref.yaml",
```

Comment:
Have you created these .json files? 
since the `make update` failed, if it would succeed these files were automatically generated for you (as well as the job under jobs/ folder).

Reply:
removed